src/script/builder/tir/block_frame.h

```diff
@@ -62,15 +64,56 @@ class BlockFrameNode : public TIRFrameNode {
 };
 
 class BlockFrame : public TIRFrame {
+ public:
+  BlockFrame() = default;
```

Comment:
why do we need this method? (probably not)

Reply:
Here my point is to pass the `make_object<PrimFuncFrameNode>` as `PrimFuncFrameNode` has field `BlockFrame root_block_frame`([here](https://github.com/cyx-6/tvm/blob/eff446bccdd085ad888ee8bdc1e009ac1927619c/src/script/builder/tir/prim_func_frame.h#L38)). Maybe there are better alternatives.

Reply:
I see. Usually we do:

```C++

class PrimFuncFrameNode : ... {
  ...
  BlockFrame root_block_frame{nullptr};
};
```